roles/initialize_disks/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Should we not leave a "lorem ipsum" file, at least we can add  "TBD" or something

Reply:
I've updated it to reflect better the purpose of the role